src/assets/styles/other/_tn-styles.scss

```diff
@@ -570,6 +570,11 @@ $primary-dark: darken(map-get($md-primary, 500), 8%);
   }
   /////////////////// END SpecSelectors
 
+  .cdk-overlay-pane {
```

Comment:
This is a very generic selector. To avoid issues with styles throughout the system unintended, we should use a more specific css selector which only applies to where we want.